sml/src/sml_list.c

```diff
@@ -171,11 +171,11 @@ sml_list *sml_list_entry_parse(sml_buffer *buf, struct workarounds *workarounds)
 	/*
 	 * Work around DZG meter - it encodes the consumption wrong:
 	 * The value uses a scaler of -2, so e.g. 328.05 should be
-	 * encoded as an unsigned int:
+	 * encoded as an unsigned int with 2 bytes (called Unsigned16 in the standard):
 	 *   63 80 25 (0x8025 == 32805 corresponds to 328.05W)
-	 * or as a signed int:
+	 * or as an unsigned int with 3 bytes (not named in the standard):
 	 *   64 00 80 25
```

Comment:
It's possible I meant "as a signed int" but meant to write "54 00 80 24" on line 177

but this is consistent, looks good to me.

Reply:
I added your example as yet another valid alternative to the comment (important because this is how it should be done). The type of the power value should be signed because it can get negative (if energy is produced on the consumer side).

Reply:
Well, I think it's probably valid to sometimes use unsigned and signed for the same entry in different messages? So I'm not sure "can get negative" matters much as long as it's not negative right now.

Reply:
You are probably right. Anyway, now we have comments that provide several choices of correct encodings.